src/lib/components/css/table.css

```diff
@@ -10,7 +10,7 @@ table:not(.table) {
 	--table-letter-spacing: var(--f-ui-md-spacing, normal);
 	border-collapse: separate;
 	border-spacing: 0 var(--border-spacing);
-	margin-top: calc(-1 * var(--border-spacing));
+	margin: calc(-1 * var(--border-spacing)) 0;
```

Comment:
@kenkunz so this one issue is caused by the overflow property being affected by negative margins. Addressed this one here — 3e04d867ab73716f787bd1c3c28fcabbe935740d

Reply:
The **Transactions** table on **Trade details** page has the same overflow issue.

Rather than having to fix this with custom overflow styles every time we use `DataTable`, what do you think of adding the required overflow styles in the `DataTable` component itself?